{"name": "test_wrong_pwd", "status": "passed", "description": "密码输入6次错误", "attachments": [{"name": "log", "source": "1fd3be07-6474-492a-afe6-8c6bdbcd07aa-attachment.txt", "type": "text/plain"}, {"name": "stdout", "source": "30001681-9e45-44e8-be80-f7654196e1d5-attachment.txt", "type": "text/plain"}, {"name": "stderr", "source": "4ac8e4aa-a962-41db-97eb-17461ecdf1d8-attachment.txt", "type": "text/plain"}], "start": 1631069407138, "stop": 1631069408324, "uuid": "f0a1b231-2211-48f0-8a26-7447ba4e7faa", "historyId": "7b9480ebb1961c7c14cb69cc7e65ba50", "testCaseId": "e4c5762512a76c681290f066bcc0e861", "fullName": "test_case.Login_case.TestLogin#test_wrong_pwd", "labels": [{"name": "feature", "value": "登录模块"}, {"name": "story", "value": "参数填写不正确，登录失败"}, {"name": "severity", "value": "critical"}, {"name": "parentSuite", "value": "test_case"}, {"name": "suite", "value": "Login_case"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "DESKTOP-KSOAST3"}, {"name": "thread", "value": "13084-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_case.Login_case"}]}